modules/library/src/main/scala/zio/elasticsearch/package.scala

```diff
@@ -67,6 +60,18 @@ package object elasticsearch extends IndexNameNewtype with RoutingNewtype {
     def aggregations: RIO[R, Map[String, AggregationResult]] =
       zio.flatMap(_.aggregations)
 
+    /**
+     * Executes the [[ElasticRequest.SearchRequest]] or the [[ElasticRequest.SearchAndAggregateRequest]].
+     *
+     * @param name
+     *   the name of the aggregation to retrieve
+     * @return
+     *   a [[RIO]] effect that, when executed, will produce the aggregation as instance of
+     *   [[result.AvgAggregationResult]].
```

Comment:
Check if this compiles without warnings.

Reply:
I don't see any warning regarding RIO, only a couple regarding unused methods.